### ISSUE TITLE
Scale table 

### ISSUE THREAD

**Issue by rzr** [rzr/pinball-table-hurd#9]
It would be good to have table aligned to other for pincab mode:

https://purl.org/rzr/pinball#

![seco-b68](https://files.mastodon.social/media_attachments/files/112/016/413/339/201/838/original/fe4b586561531ba5.jpg)

![](https://files.mastodon.social/media_attachments/files/106/303/780/464/825/504/original/f51f6dc7e986669a.png)

**Comment by rzr** [comment]
I tried a bit but it's not easy, since some elements are "outside" the base

@paolo-caroni which editor did you use ?

**Comment by paolo-caroni** [comment]
This will be very difficult, honestly I have designed the table for the top view.
I have used pinedit qt3 version for vector structure and gimp for the pictures.

**Comment by rzr** [comment]
Is there any noticable difference between qt3 and qt4 version ?

If you still have this editor would you mind to share a screenshot of the loaded table ?

BTW congratulation you're contributed to debian this new table:

https://tracker.debian.org/pkg/pinball-table-hurd

I was even able to show it earlier today at some debian online event, you can rewatch it at:

http://pinball.sf.net

Stay tuned:

https://purl.org/rzr/pinall

I plan to realease a new hurd table for SDL-2 

Some updates will be welcome, would you like to help ?

Like I am sure music can be replaced... :)

**Comment by paolo-caroni** [comment]
> Is there any noticable difference between qt3 and qt4 version ?
>
I don't know, i'm not sure, because I never use the qt4 version.
> 
> If you still have this editor would you mind to share a screenshot of the loaded table ?
>
I don't have it anymore, maybe i have an early stage screenshot, but I have to verify on my old computer.
> 
> BTW congratulation you're contributed to debian this new table:
> 
> https://tracker.debian.org/pkg/pinball-table-hurd
> 
Thank you, I'm very proud of it.

> I was even able to show it earlier today at some debian online event, you can rewatch it at:
> 
> http://pinball.sf.net
> 
> Stay tuned:
> 
> https://purl.org/rzr/pinall
> 
> I plan to realease a new hurd table for SDL-2
>
So do you want to change the emilia pinball code?
> 
> Some updates will be welcome, would you like to help ?
>
Honestly actually i'm in to other projects and I don't have so much time to give a contribute to this one.
I'm sorry I hope you will understand.
> 
> Like I am sure music can be replaced... :)
>
Yes sure, could be replaced years ago but I had problems about music licence.

**Comment by rzr** [comment]
FYI pinedit is now usable as snap maybe I need to document how to use and install it:

https://adoptware.github.io/pinball/editor

https://snapcraft.io/pinedit

Is anyone interested ?

Let me know

https://mastodon.social/@rzr/106303788367685056


![](https://files.mastodon.social/media_attachments/files/106/303/780/464/825/504/original/f51f6dc7e986669a.png)

**Comment by jgengelhardt** [comment]
I'm interested! However, my Linux machine is a Raspberry Pi, and there isn't an ARM-compatible Snap package for the table editor currently. I've been trying to compile from source but have run into issues.

**Comment by rzr** [comment]
maybe you could make it run in a docker container too

ping me at:

https://mastodon.social/@rzr/107066163766015722

**Comment by jgengelhardt** [comment]
On an Ubuntu laptop now. When I go to open a table, I get the following error. When I click to accept or close out, it doesn't close, and I have to kill it from the terminal.
![Screenshot from 2022-09-26 13-57-52](https://user-images.githubusercontent.com/44878057/192357996-8a884759-033c-4cca-887d-8a32110dbcc8.png)

**Comment by rzr** [comment]
Are you running it from snap package ?

**Comment by jgengelhardt** [comment]
Yes, here's the snap list info: pinedit  0+git.4aae050  1  latest/beta  rzr  -
Running on Ubuntu 22.04.1 LTS

**Comment by rzr** [comment]
it worked for me:

```
cat /etc/os-release 
PRETTY_NAME="Ubuntu 22.04.1 LTS"

sudo snap install pinedit --edge
# pinedit            0.3.20210520+git2.09e0aa9   4      latest/edge    rzr           -

groups $USER
rzr : rzr adm dialout cdrom sudo audio dip plugdev staff kvm lpadmin sambashare admin docker
```

**Comment by jgengelhardt** [comment]
Removing pinedit then installing with ```sudo snap install pinedit --edge``` solved the problem!

**Comment by rzr** [comment]
feel free to file issue on pinedit and good luck, 
sorry If i am slow at providing support, but I'll release a new version of pinball with your patches this year
